src/components/InputDeviceConfig/AudioVideo.vue

```diff
@@ -71,13 +88,17 @@ export default {
   data() {
     return {
       iVTitle: "Media Input Config",
-      iVBody:
-        "Configure your audio and video inputs here. Click Renumerate Devices to scan for new sources.",
+      iVBody: `Configure your audio and video inputs here. Click "Re-scan Devices" to scan for new sources.`,
       switchingAudio: false,
-      switchingVideo: false
+      switchingVideo: false,
+      gainRangeValue: 1
```

Comment:
Using a value on data so we can keep track of what the value is doing.
Vue wasn't aware of the value before this.

We also update the value when the component is created below in the `created` function.